test/utils/CommonActions.js

```diff
@@ -52,14 +51,6 @@ class CommonActions {
 
     }
 
-    /**
-     * Method to wait element has been enable.
-     * @param elementCSS WebElement locator.
-     */
-    static waitElementToReady(elementCSS) {
```

Comment:
Maybe you need check this is to improve stability.

Reply:
sure it is not neccesary.I tried it